hc-dna/default.nix

```diff
@@ -0,0 +1,33 @@
+let
+  holonixPath = builtins.fetchTarball {
+    url = "https://github.com/holochain/holonix/archive/55a5eef58979fb6bc476d8c3e0c028cdeb1b5421.tar.gz";
+    sha256 = "sha256:0q6d0rql1pyy93xw1c8s28jjjcgk1zgwxwixsp9z5r4w2ihaz3zg";
+  };
+  holonix = import (holonixPath) {
+    includeHolochainBinaries = true;
+    holochainVersionId = "custom";
+
+    holochainVersion = {
+      rev = "d003eb7a45f1d7125c4701332202761721793d68";
+      sha256 = "0qxadszm2a7807w49kfbj7cx6lr31qryxcyd2inyv7q5j7qbanf2";
+      cargoSha256 = "sha256:129wicin99kmxb2qwhll8f4q78gviyp73hrkm6klpkql6810y3jy";
+      bins = {
+        holochain = "holochain";
+        hc = "hc";
+        kitsune-p2p-proxy = "kitsune_p2p/proxy";
+      };
+
+      lairKeystoreHashes = {
+        sha256 = "0khg5w5fgdp1sg22vqyzsb2ri7znbxiwl7vr2zx6bwn744wy2cyv";
+        cargoSha256 = "sha256:1lm8vrxh7fw7gcir9lq85frfd0rdcca9p7883nikjfbn21ac4sn4";
+      };
+    };
+    holochainOtherDepsNames = ["lair-keystore"];
+  };
+  nixpkgs = holonix.pkgs;
+in nixpkgs.mkShell {
+  inputsFrom = [ holonix.main ];
+  buildInputs = with nixpkgs; [
+    binaryen
+  ];
+}
```

Comment:
There is another default.nix in the root of this repo, should we instead update that one?

Reply:
I just remember that I move the default.nix from hc-dna to root for setting up the CI. I will update this PR later and gets it merge.